### ISSUE TITLE
use AccordionBox

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/graphing-lines#106]
Graphing Lines currently does not use AccordionBox, it uses a Panel + ExpandCollapseButton to create something that looks like an AccordionBox (see `EquationControls`).  This should be changed to take advantage of phet-io and a11y instrumentation of AccordionBox.

**Comment by pixelzoom** [comment]
Done, I was able to convert this with no changes to spacing, margins, etc.   Closing.
